pandas/tests/frame/indexing/test_xs.py

```diff
@@ -297,3 +297,18 @@ def test_xs_levels_raises(self, klass):
         msg = "Index must be a MultiIndex"
         with pytest.raises(TypeError, match=msg):
             obj.xs(0, level="as")
+
+    def test_xs_droplevel_false(self):
+        # GH: 19056
+        mi = MultiIndex.from_tuples(
+            [("a", "x"), ("a", "y"), ("b", "x")], names=["level1", "level2"]
+        )
+        df = DataFrame([[1, 2, 3]], columns=mi)
+        result = df.xs("a", axis=1, drop_level=False)
```

Comment:
do we have a test for Series with drop_level=False?

Reply:
``axis=1`` does not work for a Series either way, doesn't it? Or do you mean for axis=0?

Reply:
right, for axis = 0

Reply:
Works, but no tests. Added one.